src/plugins/chessevent/templates/chessevent_event_card_block.html

```diff
@@ -1,4 +1,4 @@
-{% if admin_events_show_details %}
+{% if client.can_view_detailed_event_cards and admin_events_show_details %}
```

Comment:
```suggestion
{% if admin_events_show_details %}
```
event_admin_show_details should always be false if not `client.can_view_detailed_event_cards`. This avoids propagation in the templates and potenitial errors. Same for every `client.can_view_detailed_{tab}_cards`

Reply:
In  fact no: permissions can change while admin_xxx_show_details is and remains true, so both tests are needed.